### PR TITLE
<MediaOverlay /> - add borderRadius property

### PR DESCRIPTION
**Pull request by tzachvw**
#pr

### CODE REVIEW

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_7028/

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_7028

**Review by milkyfruit — APPROVED** [review]
works great!

File: packages/wix-style-react/src/MediaOverlay/MediaOverlay.js [hunk 1]
```diff
@@ -59,6 +59,9 @@ class MediaOverlay extends React.PureComponent {
 
     /** clear borders radius when displayed in sharp-edges containers */
     removeRoundedBorders: PropTypes.bool,
+
+    /** Border radius of the Media Overlay element box. */
+    borderRadius: PropTypes.string,
```
Comment (mykas) [inline packages/wix-style-react/src/MediaOverlay/MediaOverlay.js:64]: Lets support both: string & number

File: packages/wix-style-react/src/MediaOverlay/index.d.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ export interface MediaOverlayProps {
   onClick?(): void;
   removeRoundedBorders?: boolean;
   className?: string;
+  borderRadius?: string;
```
Comment (mykas) [inline packages/wix-style-react/src/MediaOverlay/index.d.ts:14]: Lets support both: string & number

**Review by mykas — APPROVED** [review]
Thank you @tzachvw !

**Pull request merged**